polkadot-protocol/protocol-components/cryptography.md

```diff
@@ -0,0 +1,116 @@
+
+## Digital signatures
+
+Digital signatures are a way of verifying the authenticity of a document or message using asymmetric keypairs. They are used to ensure that a sender or signer's document or message has not been tampered with in transit, and for recipients to verify said data is accurate and from the expected sender.
+
+Signing digital signatures only requires a low level understanding of mathematics and cryptography. For a conceptual example -- when signing a check, it is expected that the check cannot be cashed multiple times. This is not a feature of the signature system, but rather the check serialization system. The bank will check that the serial number on the check has not already been used. Digital signatures essentially combines these two concepts, allowing the signature itself to provide the serialization via a unique cryptographic fingerprint that cannot be reproduced.
+
+Unlike with a pen and paper signatures, knowledge of the digital signature cannot be used to create other signatures. Digital signatures are often used in bureaucratic processes, as they are more secure than simply scanning in a signature and pasting it onto a document.
+
+Polkadot SDK provides multiple different cryptographic schemes and is generic such that it can support anything which implements the [`Pair` trait](https://paritytech.github.io/polkadot-sdk/master/sp_core/crypto/trait.Pair.html){target=\_blank}.
+
+### Example of Creating a Digital Signature
+
+1. The sender creates a hash of the message
+2. The hash is encrypted using the sender's private key, creating the signature
+3. The message and signature are sent to the recipient
+4. The recipient decrypts the signature using the sender's public key
+5. The recipient hashes the received message and compares it to the decrypted hash
+
+If the hashes match, the signature is valid, confirming the message's integrity and the sender's identity
+
+## Elliptic Curve
+
+Blockchain technology requires the ability to have multiple keys creating a signature for block proposal and validation. To this end, Elliptic Curve Digital Signature Algorithm (ECDSA) and Schnorr signatures are two of the most commonly used methods. While ECDSA are a far simpler implementation, Schnorr signatures are more efficient when it comes to multi-signatures.
+
+Schnorr signatures bring some noticeable features over the ECDSA/EdDSA schemes:
+
+- It is better for hierarchical deterministic key derivations
+- It allows for native multi-signature through [signature aggregation](https://bitcoincore.org/en/2017/03/23/schnorr-signature-aggregation/){target=\_blank}
+- It is generally more resistant to misuse
+
+One sacrifice that is made when using Schnorr signatures over ECDSA is that both require 64 bytes, but only ECDSA signatures communicate their public key.
```

Comment:
Nit picking here, but I'd briefly explain why the requirements of 64 bytes is a sacrifice. Probably, you can add something like:

"One sacrifice when using Schnorr signatures over ECDSA is that both require 64 bytes. This can potentially increase the total communication overhead, but only ECDSA signatures communicate their public key."